CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
-## 1.0.1 (upcoming) - HA-core fixtures and error notification fix
+## 1.1.0 - HA-core fixtures and error notification fix
```

Comment:
"## 1.1.0 - Add HA-core test-fixtures, Plugwise notifications, and improve error-detection"

Reply:
Done